editor/editor_help.cpp

```diff
@@ -101,7 +101,7 @@ static bool _attempt_doc_load(const String &p_class) {
 // Removes unnecessary prefix from p_class_specifier when within the p_edited_class context
 static String _contextualize_class_specifier(const String &p_class_specifier, const String &p_edited_class) {
 	// If this is a completely different context than the current class, then keep full path
-	if (!p_class_specifier.begins_with(p_edited_class)) {
+	if (!p_class_specifier.begins_with(p_edited_class + ".")) {
```

Comment:
Doesn't that obsolete the whole `if` block on line 108?

Reply:
Testing some things here - it was fine from the outer class but yall are right, I'm checking inner classes and that does a few things. I'll mark this as draft until I've more thoroughly tested it :)

Note to self, do not work on PRs at 6am 😅

Reply:
I believe I have addressed the issue but please do double-check my logic 😴